src/main/kotlin/no/nav/emottak/eventmanager/service/EventService.kt

```diff
@@ -1,13 +1,22 @@
 package no.nav.emottak.eventmanager.service
 
+import kotlinx.serialization.json.Json
 import no.nav.emottak.eventmanager.log
+import no.nav.emottak.eventmanager.persistence.repository.EventsRepository
+import no.nav.emottak.utils.events.model.Event
+import kotlin.uuid.ExperimentalUuidApi
 
-class EventService {
+@OptIn(ExperimentalUuidApi::class)
```

Comment:
Et lite triks for å slippe å gjenta disse `@OptIn` gjennom kodebasen er å konfigurere dette i Gradle. Ref: https://github.com/navikt/emottak-utils/blob/44a21fe956474dae273100d6e370b8201a4b9ed4/build.gradle.kts#L17